src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -119,6 +119,14 @@ module sonic-device_metadata {
                     default enable;
                 }
 
+                leaf yang_config_validation {
+                    type enumeration {
```

Comment:
Can you use mode-status instead?
https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-yang-models/yang-templates/sonic-types.yang.j2#L178

Reply:
Changed to use stypes:mode-status